fastchat/serve/gradio_web_server.py

```diff
@@ -166,11 +172,12 @@ def get_model_list(controller_url, register_api_endpoint_file, multimodal):
     if register_api_endpoint_file:
         api_endpoint_info = json.load(open(register_api_endpoint_file))
```

Comment:
I haven't tested all endpoints, but if you have tested them, I think the other parts look good to me. One thing remaining is to have a default value for the fields in `api_endpoints.json`. Make sure "vision-arena" is read as false when the field is missing in the JSON file.

Reply:
Oh, I saw you added a default value for "vision-arena" and "text-arena". It would be better to have a more systematic way, but for now, it's fine for the PR to go!